cedar-policy-validator/src/schema.rs

```diff
@@ -658,7 +792,17 @@ impl ValidatorSchema {
         // Pass `true` here so that we also check that the action hierarchy does
         // not contain cycles.
         compute_tc(&mut action_ids, true)?;
-
+        #[cfg(feature = "extended-schema")]
+        let common_type_validators = common_types
+            .clone()
+            .into_iter()
+            .filter(|ct| {
+                // Exclude primitives since we only want the definition locations of common types that the schema introduces
+                !matches!(ct.1.ty, Type::ExtensionType { .. })
+                    && !matches!(ct.1.ty, Type::Primitive { .. })
```

Comment:
what if the user writes `type T = String;`?

Reply:
Oh this is a good callout, will fix this